admin-manual/installation/upgrading.rst

```diff
@@ -368,9 +368,57 @@ Upgrading with a custom theme plugin
 If you have developed a custom theme plugin for your application (for more
 information, see :ref:`customization-custom-theme`), you may need to perform
 an additional step following an upgrade to ensure that all pages are styled
-correctly. Please note that the instructions below are for upgrading a custom
-theme that continues to the Bootstrap 2 (“BS2”). Documentation for upgrading to
-BS5 will be provided at a later date.
+correctly.
+
++++++++++++
+Bootstrap 5
++++++++++++
+
+Additional steps for tarball installations:
+*******************************************
+
+If not already installed, first `download the node.js binary distributuion
+<https://nodejs.org/en/download>`_ and export the PATH variable.
+
+The tarball is missing two required files for this: copy the
+`package.json <https://github.com/artefactual/atom/blob/stable/2.7.x/package.json>`_
```

Comment:
Also here.

Reply:
Hmm, good call. 

While making changes, I just realized that this link is to our stable/2.7.x repository - but if we don't want to have to update this GitHub link with every release, I would suggest that you update that link and the one in the other file to: 

```
`package.json <https://github.com/artefactual/atom/blob/HEAD/package.json>`_

```

Using `HEAD` will ensure that the link always points to the latest commit on the default branch, kind of like using `/latest/` in our documentation web links instead of a version number!

Reply:
We don't want them to use the qa/2.x version of those file in the stable release. Maybe this could be solved using the version var (from this docs) in the URL so it's updated based on that?